spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,16 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    it 'returns http success' do
+      get '/users/:user_id/posts'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/:user_id/posts/:id'
+      expect(response).to have_http_status(:success)
+    end
   end
```

Comment:
- [ ] @ritikarawat220 great work adding test for the index and show actions, however you forgot to add tests to show `a correct template was rendered.` and `the response body includes correct placeholder text` for both index and show actions, kindly include the tests, they are part of project requirements as shown below
![image](https://github.com/ritikarawat220/Blog-App/assets/61408860/d71c223e-bd18-45f3-b9e8-78dbbb33185c)